### ISSUE TITLE
Link text should have hover and focus states

### ISSUE THREAD

**Issue by mcroud** [mozilla/community-portal#342]
![Untitled7](https://user-images.githubusercontent.com/42435754/75663792-4fe5c080-5c69-11ea-84c3-d35bac656461.gif)

Currently keyboard users can not see where they are on the page, can we have normal link text turn black (including the underline) when it is hovered over or focused.
